index.html

```diff
@@ -0,0 +1,237 @@
+            <h3 class="program-text1">CC Ignite</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum.
+            </p>
+          </div>
+          <a href="#" class="join-btn">Join CC Global Submit 2015</a>
+        </div>
+        <a href="#" class="program-link">SEE THE WHOLE PROGRAM</a>
+      </section>
+
+      <!-- Speakers -->
+      <section class="speakers">
+        <h2>featured Speakers</h2>
+        <hr class="line1" />
+        <div class="speakersContainer">
+        </div>
+        <button class="btn more">
+          <a href="#">MORE</a>
+          <img src="./img/arrow_down.png" alt="arrow icon" />
+        </button>
+        <a href="#" class="btn less">LESS</a>
+      </section>
+    </div>
+
+    <!-- menu -->
+    <section class="menu-click">
+      <div class="close-container">
+        <img class="close-icon" src="./img/cross-icon.png" alt="close icon" />
+      </div>
+      <ul class="menu-text">
+        <li>
+          <a href="#Portfolio" class="nav-link">Portfolio</a>
+          <hr class="menu-line" />
+        </li>
+        <li>
+          <a href="#about" class="nav-link">About</a>
+          <hr class="menu-line" />
+        </li>
+        <li>
+          <a href="#contact-form" class="nav-link">Contact</a>
+          <hr class="menu-line" />
+        </li>
+      </ul>
+    </section>
+
+    <!-- page 2 -->
+    <div class="page2 about-page">
+      <!-- about -->
+      <section class="about">
+        <h3 class="sub-title">"Hello! sharing the world!"</h3>
+        <h1 class="about-title">
+          WORLD CONFerance <br />
+          ALERTS 2022
+        </h1>
+        <p class="paragraph1 color">
+          Full-stack Web Development courses offered through Coursera equip learners with knowledge in building web, hybrid, and mobile solutions; understanding front-end frameworks such as Bootstrap 4 and Angular; implementing NoSQL databases using MongoDB; creating the best user
+           experience regardless of device; the basics of ...
+        </p>
+        <h2 class="contact">
+          Please contact us per Email for any further questions<br />
+          about CC Global Summi 2015!<br />
+          <a href="#">ccsummit@cckorea.org</a>
+        </h2>
+      </section>
+
+      <!-- global summit -->
+      <section class="global-summit">
+        <h2 class="">
+          Creative Commons<br />
+          Global Summit 2015 Logo
+        </h2>
+        <hr class="line1" />
+        <p class="address">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit onsectetur
+          adipisicing elit
+        </p>
+        <img
+          class="about-img"
+          src="./img/about_2013.png"
+          alt="2013 comferance image"
+        />
+        <img
+          class="about-img"
+          src="./img/about_2011.png"
+          alt="2011 comferance image"
+        />
+      </section>
```

Comment:
- [ ] Your desktop version of the 'Global Summit' section is the same as the mobile version. Kindly make sure it follows the template and has the 2 items listed as horizontal row and 2 columns.
 - Your version:
 - 
<img width="1145" alt="image" src="https://user-images.githubusercontent.com/61919617/210401837-c1ea25a2-fddd-4bb9-9271-624d4c06c88f.png">

 - Template:
 - 
<img width="1072" alt="image" src="https://user-images.githubusercontent.com/61919617/210401887-419ec82e-bd3e-476b-9c7f-438e05264f1c.png">

Reply:
done